flocker/node/agents/functional/test_cinder.py

```diff
@@ -394,3 +395,86 @@ def test_disk_attachment_fails_with_conflicting_disk(self):
                 expected_volume=attached_volume,
                 expected_status=u'in-use',
             )
+
+    @require_virsh
```

Comment:
This test doesn't require virsh. Either the decorator should be renamed, or maybe a different decorator for tests that require virtio, but not particularly virsh.